Vertexvis/vertex-web-sdk
# [PLAT-3317] Release v0.19.1

## Summary

Includes the following changes:
- Update to support the `phantom` scene view state feature in partial apply #540 
- Update docs with additional examples around `Scene` functionality #541 

## Test Plan

- Verify the release is published as expected
- Verify the newly deployed docs have more examples around `Scene` functionality

## Release Notes

N/A

## Possible Regressions

N/A

## Dependencies

N/A
